### PR TITLE
[IntelliJ] Update getting started link

### PR DESCRIPTION
**Pull request by keithchong**
Fixes https://github.com/eclipse/codewind/issues/2165

Signed-off-by: Keith Chong <kchong@ca.ibm.com>

### CODE REVIEW

**Comment by keithchong** [comment]
@jopit , please review this one too.

File: dev/src/main/java/org/eclipse/codewind/intellij/ui/constants/UIConstants.java [hunk 1]
```diff
@@ -10,11 +10,25 @@
...
+import org.eclipse.codewind.intellij.core.console.SocketConsole;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
 public class UIConstants {
+
+    @NotNull
+    public static final Map<String, Key<SocketConsole>> LOGS_KEYMAP = new HashMap<>();
```
Comment (jopit) [inline dev/src/main/java/org/eclipse/codewind/intellij/ui/constants/UIConstants.java]: This definition of `LOGS_KEYMAP` and the import statements above shouldn't be part of this change.

Comment (keithchong) [reply dev/src/main/java/org/eclipse/codewind/intellij/ui/constants/UIConstants.java]: Hmm yes indeed.

**Comment by keithchong** [comment]
Hi @jopit, it's corrected now.
